cloudferrylib/os/compute/nova_compute.py

```diff
@@ -121,15 +119,11 @@ def _read_info_resources(self, **kwargs):
         """
         Read info about compute resources except instances from the cloud.
         """
-        info = {'keypairs': {},
-                'flavors': {},
+
+        info = {'flavors': {},
                 'user_quotas': [],
                 'project_quotas': []}
 
-        kps = keypairs.DBBroker.get_all_keypairs(self.cloud)
```

Comment:
But we do not use this info anywhere in the code... I suppose all such stuff done here:
https://github.com/MirantisWorkloadMobility/CloudFerry/blob/devel/cloudferrylib/os/actions/transport_compute_resources.py#L92